QModManager/OptionsManager.cs

```diff
@@ -60,7 +63,63 @@ internal static void Postfix(uGUI_OptionsPanel __instance)
                     AddToggleOption.Invoke(__instance, new object[] { ModsTab, "Enable debug logs", Config.EnableDebugLogs, new UnityAction<bool>(value => Config.EnableDebugLogs = value), null });
                     AddToggleOption.Invoke(__instance, new object[] { ModsTab, "Enable developer mode", Config.EnableDevMode, new UnityAction<bool>(value => Config.EnableDevMode = value), null });
                 }
+                #endregion Mod Config
 
+                #region Mod List
+                //Create new Tab in the Menu
+                ModListTab = __instance.AddTab("QMods List");
+
+                //Add QMM Informations
+#if SUBNAUTICA_EXP || BELOWZERO_EXP
+                __instance.AddHeading(ModListTab, $"QModManager -Experimental- running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
+#else
+                __instance.AddHeading(ModListTab, $"QModManager running version {Assembly.GetExecutingAssembly().GetName().Version.ToStringParsed()}");
+#endif
+                //Add SML Informations
+                var modprio_sml = QModServices.Main.GetMod("SMLHelper");
+                if (modprio_sml != null)
+                {
+                    __instance.AddHeading(ModListTab, $"{modprio_sml.DisplayName} {(modprio_sml.Enable ? $"v{modprio_sml.ParsedVersion}" : $"")} {(modprio_sml.IsLoaded ? "is activated" : "is deactivated")}");
+                }
+                else
+                {
+                    __instance.AddHeading(ModListTab, $"SML Helper is not installed");
```

Comment:
Remove the whitespace that's separating `SML` from `Helper`:
```suggestion
                    __instance.AddHeading(ModListTab, $"SMLHelper is not installed");
```